flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/codegen/CodeGeneratorContext.scala

```diff
@@ -500,7 +527,7 @@ class CodeGeneratorContext(val tableConfig: TableConfig) {
     * Adds a reusable TimeZone to the member area of the generated class.
     */
   def addReusableTimeZone(): String = {
```

Comment:
I think it should read `tableConfig.getTimeZone` and used by next support for `Timestamp with local zone`.
`addReusableLocalDateTime` should just read `TimeZone.getDefault`.

Reply:
@JingsongLi Maybe we should update `TableConfig` and use `ZoneId` instead of the old `TimeZone` class. I haven't looked in to the details but I read: "A {@code ZoneId} is used to identify the rules used to convert between
an {@link Instant} and a {@link LocalDateTime}."

Reply:
@twalthr good suggestion, +1, `ZoneId` for java 8 classes.
`TimeZone` is an outdated class and troublesome.